src/PowerShellEditorServices/Extensions/EditorWindow.cs

```diff
@@ -3,16 +3,21 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
-namespace Microsoft.PowerShell.EditorServices.Services.PowerShellContext
+using OmniSharp.Extensions.LanguageServer.Protocol.Server;
+using System.Threading.Tasks;
+
+namespace Microsoft.PowerShell.EditorServices.Extensions
 {
     /// <summary>
     /// Provides a PowerShell-facing API which allows scripts to
     /// interact with the editor's window.
     /// </summary>
-    public class EditorWindow
+    public sealed class EditorWindow
     {
         #region Private Fields
 
+        private readonly ILanguageServer _languageServer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_languageServer' or refactor the code to use its value.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4979332)